### DISCUSSION TITLE
Add Timestamp for Timer

### DISCUSSION THREAD

**Discussion by rozivector** [maximmax42/Discord-CustomRP#112] [Ideas]
It would be good to add feature such as customable timer at timestamp (example: 10:00:00 left). After 00:00:00 reaches, it will keep stay count to zero.

**Comment by maximmax42** [comment]
Hm. Not a bad idea, might try to implement.

**Reply by rozivector** [reply]
Alright in User Interface, there is an option Custom timestamp which is implement for elapsed time. Then there will be another an option _check box/radio button_ titled _Timer_. And if checked, the form will changed into **Custom Time** (not including date anymore) such as **XX:XX:XX** form only. If it is modified success, the presence will change into time remaining as input the timer.

P.S.: If times out, the status of timer can be hiding/set none after reach 00:00:00 from presence, you can implement it i guess.

**Reply by maximmax42** [reply]
Hm. Not sure how I would implement a time picker, but I could try.
